app/components/embedded_bindings/menu_binding/menu_binding.tsx

```diff
@@ -17,7 +17,7 @@ type Props = {
     actions: {
         doAppCall: (call: AppCallRequest, type: AppCallType, intl: any) => Promise<{data?: AppCallResponse, error?: AppCallResponse}>;
         getChannel: (channelId: string) => Promise<ActionResult>;
-        sendEphemeralPost: (message: any, channelId?: string, parentId?: string) => Promise<ActionResult>;
+        sendEphemeralPost: (message: any, channelId?: string, parentId?: string, userId?: string) => Promise<ActionResult>;
```

Comment:
I've seen this definition multiple times, perhaps add it to a `d.ts` file and use that instead of re-defining constantly?